code/_onclick/ventcrawl.dm

```diff
@@ -126,11 +126,14 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	if(!network)
 		return
 	for(var/datum/pipeline/pipeline in network.line_members)
-		for(var/atom/X in (pipeline.members || pipeline.edges))
+		for(var/X in (pipeline.members || pipeline.edges))
```

Comment:
Do these lists only have atoms for sure? I wouldn't be surprised if there are nested pipeline datums

Reply:
those lists are /obj/machinery/atmospherics/pipe so unless someone fucked up, yes